### ISSUE TITLE
Cannot auto refresh upon changes

### ISSUE THREAD

**Issue by nickypeh** [johnpapa/lite-server#56]
Hi , 
I followed the tutorial via Angular2 [5 mins quickstart](https://angular.io/docs/js/latest/quickstart.html#).
When i run using the `npm start`, it is able to run in the browser but it cannot auto refresh the browser whenever it has any changes. It only detects changes but the browser wont auto refresh. 
Is there any additional setting that i should include for the browser-sync?

Please advise.

**Comment by arproudlock** [comment]
Having the same issue.  Running node 5.9.0, npm 3.7.3.  Tried clearing cache.

**Comment by cgmartin** [comment]
Related: https://github.com/angular/angular/issues/7652

A workaround is to use an earlier version of the quickstart until this is resolved:
```
$ git clone  https://github.com/angular/quickstart  my-proj
$ cd my-proj
$ git checkout tags/2.0.0-beta.9
```

**Comment by sergiator** [comment]
same err on ubuntu 14.04 with node 4.4. My docker image for issue:
`docker pull sergiator/angular2-quickstart-lite-server-issue` 
`docker run -it -p 3000:3000 -p 3001:3001 --name lite-issue angular2-quickstart-lite-server-issue cd quickstart && npm start`
Then run from another terminal:
`sudo docker exec -i -t bash ./changer.sh`

**Comment by shakyShane** [comment]
https://github.com/angular/angular/issues/7705#issuecomment-200019429

**Comment by cgmartin** [comment]
This appears to be fixed in the latest `2.0.0.beta.12` release of the [Angular2 Quickstart](https://github.com/angular/quickstart). Closing.
